### PR TITLE
Update directory.md

### PR DESCRIPTION
**Pull request by pixelate**
- Replace http with https in links
- Remove dead links/companies
- Add Doconomy, Nextory and Qasa since they have open Rails positions

### CODE REVIEW

File: directory.md [hunk 1]
```diff
@@ -111,55 +80,51 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
...
   </li>
 
   <li>
-    <a href="http://www.magplus.com">Mag+</a>
+    <a href="https://www.jmkplay.se">JMK, Stockholms universitet</a>
   </li>
 
   <li>
-    <a href="http://www.mynewsdesk.com">Mynewsdesk</a>
+    <a href="https://www.magplus.com">Mag+</a>
```
Comment (teoljungberg) [inline directory.md:115]: Are they still around?

File: directory.md [hunk 2]
```diff
@@ -111,55 +80,51 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
...
   </li>
 
   <li>
-    <a href="http://onegiantleap.se">One Giant Leap</a>
+    <a href="https://nextory.com">Nextory</a>
   </li>
 
   <li>
-    <a href="http://www.promoterapp.com">Promoter</a>
+    <a href="https://oktavilla.se">Oktavilla</a>
```
Comment (teoljungberg) [inline directory.md:127]: Oktavilla has closed shop

**Review by teoljungberg — APPROVED** [review]

**Comment by olleolleolle** [comment]
👋 Saying "yeah", to updating the _directory_. A cool resource.

File: directory.md (hunk 1 above)
Comment (pixelate) [reply directory.md:115]: The website is still around, but it looks like it has been bought by MPS Limited in 2016. MPS does not have any Swedish location: https://www.mpslimited.com/locations/. Should be safe to remove as well.

File: directory.md [hunk 3]
```diff
@@ -111,55 +80,51 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
...
   </li>
 
   <li>
-    <a href="http://www.jmkplay.se">JMK, Stockholms universitet</a>
+    <a href="https://www.itsinthenode.com">Its in the Node</a>
   </li>
 
   <li>
-    <a href="http://www.kollegorna.se">Kollegorna</a>
+    <a href="https://justarrived.se">Just Arrived</a>
```
[the remark this answers is not in the document]
Comment (pixelate) [reply directory.md:107]: Thanks, will remove.

File: directory.md [hunk 4]
```diff
@@ -111,55 +80,43 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
...
   </li>
 
   <li>
-    <a href="http://onegiantleap.se">One Giant Leap</a>
+    <a href="https://nextory.com">Nextory</a>
   </li>
 
   <li>
-    <a href="http://www.promoterapp.com">Promoter</a>
+    <a href="https://qasa.se">Qasa</a>
```
Comment (pixelate) [inline directory.md:119]: Discovered this as well, added now.

**Comment by olleolleolle** [comment]
@pixelate Tack, toppen!

**Pull request merged**